### PR TITLE
[GCP] Fix loadbalancing_metrics prefix

### LINKED ISSUE #7242: [O11y][GCP] Update fields in GCP Load balancing dashboards
# Description

GCP Load balancing dashboards are not being populated when integration is configured. 
The logs are being stored as `gcp.loadbalancing_metrics.*`. But dashboards are using `gcp.loadbalancing.*` field. 

Below is the PR in which the pipeline was updated.
[GCP] Add loadbalancing ingest pipeline: https://github.com/elastic/integrations/pull/4350

## Screenshot

screencapture-dev-4-kb-us-central1-gcp-cloud-es-io-9243-app-dashboards-2023-08-03-17_40_02

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR do?

Loadbalancing dashboards are using `gcp.loadbalancing` prefix but they need to use `gcp.loadbalancing_metrics` since the pipeline is using this naming. The fields prefix in `fields.yml` is also wrong.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #7242

## Screenshots

localhost_5601_app_dashboards (1)

localhost_5601_app_dashboards

### CODE REVIEW

File: packages/gcp/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.24.1"
```
Comment (bhapas) [inline packages/gcp/changelog.yml:2]: Just a thought : Should this be a minor version bump as there is a change in field name. cc : @andrewkroh

**Comment by ishleenk17** [comment]
@gpop63  What will be the impact on the users using this integration, when they upgrade?
Field names will change for them. This would be a [breaking change](https://github.com/elastic/dev/issues/2036) for them.

File: packages/gcp/changelog.yml (hunk 1 above)
Comment (ishleenk17) [reply packages/gcp/changelog.yml]: I agree, this should be a minor version bump

**Comment by gpop63** [comment]
@ishleenk17 won't be a breaking change since fields were still using the correct prefix because of the pipeline (I'll look more into it).

**Review by constanca-m — APPROVED** [review]

**Pull request merged**